airflow/migrations/versions/d38e04c12aa2_add_serialized_dag_table.py

```diff
@@ -58,13 +58,13 @@ def upgrade():
         'idx_fileloc_hash', 'serialized_dag', ['fileloc_hash'])
 
     if conn.dialect.name == "mysql":
-        conn.execute("SET time_zone = '+00:00'")
-        cur = conn.execute("SELECT @@explicit_defaults_for_timestamp")
-        res = cur.fetchall()
-        if res[0][0] == 0:
-            raise Exception(
-                "Global variable explicit_defaults_for_timestamp needs to be on (1) for mysql"
-            )
+        # conn.execute("SET time_zone = '+00:00'")
```

Comment:
I think this is a new patch we need to add, and not the one coming from 1.10.4

We should put a separate pull req for this.